packages/paste-website/src/pages/components/modal/index.mdx

```diff
@@ -0,0 +1,423 @@
+
+- Presses the Esc key
+- Presses the close "x" button in the Modal header,
+- Presses a "Cancel" button in the Modal footer
+- Clicks outside of the Modal, or
+- Performs another action that closes the Modal
+
+The Modal body is an extremely flexible container that doesn't require specific components inside of it. You can compose a Modal content area to support your use case, but elements such as Paragraph and Form Input are commonly used. See examples for common Modal patterns and dos/don'ts.
+
+### Accessibility
+
+- All elements required to interact with the modal, including closing or acknowledging it, are contained in the modal since they trap focus, and users can't interact with the underlying page.
+- Tabbing through a Modal will cycle through its content in the same way it does on a page. A Modal also supports these keyboard interactions:
+  - Escape: Closes the dialog
+- The element that serves as the modal container has a role of dialog. The dialog has either:
+  - A value set for the `aria-labelledby` property that refers to a visible dialog title.
+  - A label specified by `aria-label`.
+- The dialog container element has `aria-modal` set to `true`.
+  - This property was introduced in ARIA 1.1. As a new property, screen reader users may experience varying degrees of support for it.
+  -Applying the aria-modal property to the dialog element replaces the technique of using aria-hidden on the background for informing assistive technologies that content outside a dialog is inert. We will still hide the background content due to support inconsistencies.
+
+## Examples
+
+### Default Modal
+
+A modal is composed of:
+
+- **Header** — Headers include a Heading and close button. Heading text should be concise (2-4 words and no more than one line) and describe the information or action the modal is presenting.
+- **Body** — This container has no content requirements and allows you to compose a Modal as you need. Common components you might use include supporting body copy (with Paragraph) and form elements.
+- **Footer** — This contains all possible modal actions aligned to the right side of the modal to show users they're progressing through their task, whether that's on the parent page, on a new page, or in another step in the modal.
+
+<Callout>
+  <CalloutTitle>Provide an accessible label</CalloutTitle>
+  <CalloutText>
+    Be sure to assign the aria-labelledby property on the modal container to the id if the modal heading.
+  </CalloutText>
+</Callout>
+
+A Modal has a default width of 608 px to allow for an optimal reading length at 70-75 characters per line at default Paragraph size at 100% magnification. At viewports smaller than 608 px, the Modal will fill the width of the viewport and pin to the bottom. The Modal will grow in height with the amount of content in it. Once the modal reaches 90% of the height of the viewport, the body will begin to scroll.
```

Comment:
This px value might be wrong now that we're using a token value.

Reply:
I made sure to update them to the token sizes in pixels. 👍